packages/react/src/components/participant/AudioTrack.tsx

```diff
@@ -0,0 +1,64 @@
+import { Participant } from 'livekit-client';
+import * as React from 'react';
+import { useMediaTrackBySourceOrName } from '../../hooks';
+import { AudioSource, log } from '@livekit/components-core';
+import { useEnsureParticipant } from '../../context';
+import { RemoteAudioTrack } from 'livekit-client';
+
+export type AudioTrackProps<T extends HTMLMediaElement = HTMLMediaElement> =
+  React.HTMLAttributes<T> &
+    ({ source: AudioSource; name?: undefined } | { name: string; source?: undefined }) & {
+      participant?: Participant;
+      onSubscriptionStatusChanged?: (subscribed: boolean) => void;
+      volume?: number;
```

Comment:
Maybe a small docstring that tells the user which range is required. I guess `0-1`?

Reply:
generally it's 0-1, however users have the possibility to use the webAudio option in which case the range can also be user defined. Will add a docstring that says `by default the range is between 0 and 1`